tutorials/glod-game/steps/step-3.md

```diff
@@ -999,408 +1000,409 @@ To do this we will:
 
 === "Code Added"
     
-    ```{.clojure .annotate linenums="1"}
-    0 >= .character-direction ;; 0 = facing left, 1 = facing right ;; (1)
-    ```
+  ```{.clojure .annotate linenums="1"}
+  0 >= .character-direction ;; 0 = facing left, 1 = facing right ;; (1)
 
-    1. Added to line 12 under `initialize-character`.
```

Comment:
This wasn't meant to be in the code block I believe (it is now)

Reply:
This has been fixed in the latest commit [e1f3e72](https://github.com/fragcolor-xyz/shards-examples/pull/67/commits/e1f3e7242832cb0fae5a88b91ad51b076aec2a78), there were missing quotes to close off the preceding code block and open the proceeding one.